packages/swarm-components/package.json

```diff
@@ -49,12 +49,12 @@
     "inert": "5.1.3",
     "jest": "24.7.1",
     "jest-image-snapshot": "2.8.1",
-    "puppeteer": "1.14.0"
+    "puppeteer": "1.14.0",
+    "swarm-icons": "^3.8.354"
```

Comment:
`swarm-icons` is defined here as a dev dependency as well as in the prod dependencies section. It should be one or the other, not both.